0009-non-helium-hotspots.md

```diff
@@ -0,0 +1,239 @@
+radio chip.
+
+However, even with this scheme we still need a way to allow Hotspots not manufactured by Helium Inc or authorized
+manufacturers to not only participate in PoC but also be candidates for consensus membership.
+
+To counter that and allow any hotspot to participate in PoC and have consensus membership candidacy, we propose to
+introduce "Levels of Trust".
+
+## Current Network Behavior
+
+In the current network, onboarding a Helium Inc manufactured Hotspot grants full Proof-of-Coverage priviledges. 
+
+As mentioned in the problem defintion, we cannot assume that every single DIY hotspot is going to incorporate a GPS chip
+or a radio chip.  Malicious enttities may try to game the system by tweaking/removing hardware and yet be able to
+participate in PoC. Currently we _know_ that every single hotspot being added to the network is manufactured by helium
+and has the required hardware to participate in proof-of-coverage challenges, as soon as we allow other hotspots to join
+the network we have no verifiable way of being able to separate a virtual network from a real one.
+
+To mitigate that, we propose a new model for establishing trust among hotspots and subsequently change how hotspots earn
+HNT, perhaps aptly named "Levels of Trust".
+
+## Levels of Trust
+
+We introduce the concept of Levels of trust, defining constraints, relationships and progress of what it means for a
+hotspot to be in a particular Level and subsequently transition between Levels. This is akin to a character Leveling up
+in a game, except the characters are hotspots!
+
+Below is a visual representation of the aforementioned Levels, the most important takeaway here is that any higher Level
+encompasses all the benefits and constraints of the lower Levels (with a few exceptions).
+
+                                                        +------------+
+                                                        |            |
+                                                +-------+  Level 3A  +---------+
+                                                |       |            |         |
+    +-----------+         +-----------+         |       +------------+         |         +-----------+
+    |           |         |           |         |                              |         |           |
+    |  Level 1  +-------->+  Level 2  +-------->+                              +-------->+  Level 4  |
+    |           |         |           |         |                              |         |           |
+    +-----------+         +-----------+         |       +------------+         |         +-----------+
+                                                |       |            |         |
+                                                +-------+  Level 3B  +---------+
+                                                        |            |
+                                                        +------------+
+
+### Level I: Getting on the network
```

Comment:
it might be easier for people to remember this stuff if you were to give the levels descriptive names instead of 1-5.

Reply:
L1: Observer
L2: Participant
L3: Challenger
L4: Consensus Member
????????

Reply:
`L4: Consensus Member` -> `L4: Consensus Candidate` :)